.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "asf4"]
 	path = asf4
-	url = https://github.com/adafruit/asf4
+	url = https://github.com/lancaster-university/asf4
```

Comment:
Let’s make sure to send them a PR upstream

Reply:
No, it's a fix they applied to fix timeouts on I2C devices that stretch the clock longer than a few seconds. The patch reduces the time to ~650 us rather than 4 seconds, they won't want it